fuzzy-norms/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "Fuzzy Logic norms",
-  "description": "Computing fuzzy-logic T-norms and T-conorms as new dataset features",
+  "description": "In this package, you will find some utilities to apply fuzzy logic\nt-norms and t-conorms to datasets.\n\nYou can find all the information about the T-norms and T-conorms in\nthe following links:\n\n\n- [T-norms and T-conorms](https://en.wikipedia.org/wiki/T-norm)\n\n - [Construction of T-norms](https://en.wikipedia.org/wiki/Construction_of_t-norms)\n\nFields used as operands must contain **real values between 0 and\n1**. As they are logical values it doesn't make sense having values\noutside this range. In this case, the script will raise an error.\n\nThe script will create a new column in the dataset for each one of the\nnorms specified in the `norms` list.\n\n\n**T-norms and T-conorms**\n\nBellow, you can find an example of the content of the `norms` input:\n\n\n[\"min-tnorm\",\"product\",\"lukasiewicz\", \"drastic-tnorm\",\n\"nilpotent-min\",[\"schweizer-sklar\", 0.5], [\"hamacher\", 0,5]]\n\n\nAs you can see there are some norms that need a parameter, that has to\nbe specified in this way.\n\nThese are the possibles t-norms:\n\n - **min-tnorm**: Minimum t-norm. Also called the Gödel t-norm. No\n   parameters.\n - **product**: Product t-norm. The ordinary product of real\n   numbers. No parameters.\n - **lukasiewicz**: Łukasiewicz t-norm.  No parameters.\n - **drastic-tnorm**: Drastic t-norm.  No parameters.\n - **nilpotent-min**: Nilpotent minimum t-norm. No parameters.\n - **[schweizer-sklar p]**: Schweizer–Sklar t-norms. Parameter *p* in\n   the range [-∞, ∞]\n - **[hamacher p]**: Hamacher t-norms. Parameter *p* in the range [0,\n   ∞]\n - **[yager p]**: Yager t-norms. Parameter *p* in the range [0,\n   ∞]\n - **[aczel-alsina p]**: Aczél–Alsina t-norms. Parameter *p* in the range [0,\n   ∞]\n - **[dombi p]**: Dombi t-norms. Parameter *p* in the range [0,\n   ∞]\n - **[sugeno-weber p]**: Sugeno–Weber t-norms. Parameter *p* in the range [-1,\n   ∞]\n\n\nThese are the possible t-conorms:\n\n - **max-tconorm**: Maximum t-norm. Dual to the minimum t-norm, is the\n   smallest t-conorm. No parameters\n - **probabilistic**: Probabilistic t-norm. It's dual to the product\n   t-norm. No parameters.\n - **bounded**: Bounded t-norm. It'ss dual to the Łukasiewicz t-norm. No\n   parameters.\n - **drastic-tconorm**: Drastic t-conorm. It's dual to the drastic\n   t-norm. No paramters.\n - **nilpotent-max**: Nilpotent maximum t-conorm. It's dual to the\n   nilpotent minumum.\n - **einstein-sum**: Einstein t-conorm. It's a dual to one of the Hamacher t-norms. \n\nFind more info in [readme](https://github.com/whizzml/examples/blob/master/fuzzy-norms/readme.md)",
```

Comment:
Bellow -> Below

As you can see there are some norms that need a parameter, that...-> As you can see, there are some norms that need a parameter that...

It'ss dual -> It's dual

nilpotent minumum -> nilpotent minimum